browser/app/js/uploads/actions.js

```diff
@@ -75,7 +75,8 @@ export const abortUpload = slug => {
   }
 }
 
-export const uploadFile = file => {
+export const uploadFile = (file, cb) => {
```

Comment:
`cb` and `callback` can be removed, since its not being used.

Reply:
Agree, reverted, thank you 👍